2697. Lexicographically Smallest Palindrome/solution.py

```diff
@@ -0,0 +1,20 @@
+class Solution:
+    # takes in the string s and turns it into a palindrome
+    # by replacing as few letters as possible. breaks tie by
+    # checking the lexicographical value of the current letter.
+    # returns the palindrome string.
+    def makeSmallestPalindrome(self, s: str) -> str:
+        # turn the string into a list for easier manipulation
```

Comment:
I think you can be more specific about what turning a string into a list enables.